### ISSUE TITLE
grantPermission() never prompts user on Android

### ISSUE THREAD

**Issue by signalpoint** [dpa99c/cordova-plugin-firebasex#839]
This is not a bug, just an observation that I hope will help others facing a similar issue...

When compiling for Android 13, I was not able to get `grantPermission()` to prompt the user.

After smashing my head against the wall for a couple of hours, I realized that I am using the `cordova-plugin-android-permissions` to ask for location permissions and using `grantPermission()` at the same time (because of how async works) to ask for notification permissions.

Once I stopped asking for location permissions at the same time, the user is prompted to grant notification permissions!

In conclusion, don't prompt the user to grant multiple permissions at once, you may experience unwanted results.

Thank you for this plugin, I've been using it for years. Feel free to close this "issue", thanks!
